acceptance/framework/flags/flags.go

```diff
@@ -68,13 +65,19 @@ func NewTestFlags() *TestFlags {
 	return t
 }
 
-func (t *TestFlags) init() {
-	flag.StringVar(&t.flagKubeconfig, "kubeconfig", "", "The path to a kubeconfig file. If this is blank, "+
-		"the default kubeconfig path (~/.kube/config) will be used.")
-	flag.StringVar(&t.flagKubecontext, "kubecontext", "", "The name of the Kubernetes context to use. If this is blank, "+
-		"the context set as the current context will be used by default.")
-	flag.StringVar(&t.flagNamespace, "namespace", "", "The Kubernetes namespace to use for tests.")
+type listFlag []string
+
+// String() returns a comma separated list in the form "var1,var2,var3".
+func (f *listFlag) String() string {
+	return strings.Join(*f, ",")
+}
+
+func (f *listFlag) Set(value string) error {
+	*f = strings.Split(value, ",")
+	return nil
+}
```

Comment:
Might be worth re-using existing implementation somehow: https://github.com/hashicorp/consul-k8s/blob/main/control-plane/subcommand/flags/flag_slice_value.go

Reply:
So this would be different from what we have I think, this is what it looks like using it with this type:
```
-kube-context="ctx-1" -kube-context="ctx-2"
```
vs what I have
```
-kube-contexts="ctx-1,ctx-2"
```

I think I like mine better, what do you think?

Reply:
actually this gets really messy with the namespaces and kubeconfigs, I think we should leave it with the comma separated list

Reply:
I don't have a super strong opinion between the two, but I'd prefer consistency. IMO also kubecontexts or namespaces look fine, but for kubeconfigs where you need to provide paths it doesn't look as nice.

```
-kubeconfigs=$HOME/path/to/kubeconfig1,$HOME/path/to/kubeconfig2
```

>actually this gets really messy with the namespaces and kubeconfigs, I think we should leave it with the comma separated list

Why does it get messy? They both parse into a list of strings I believe.

Reply:
Not messy in the sense of the algorithm, because like you said it stays the same, I just mean UX wise for the user:
-kube-context="ctx-1" -kube-context="ctx-2" -kube-namespace="ns1" -kube-namepsace="ns2"

Maybe it's not that bad, but you have to know that the first `-kube-context` maps to the first `-kube-namepsace`

Mind you maybe it makes it easier if used in this case:
-kube-context="ctx-1" -kube-namespace="ns1" -kube-context="ctx-2" -kube-namespace="ns2"

Reply:
> Maybe it's not that bad, but you have to know that the first -kube-context maps to the first -kube-namespace

You need to know that with comma separated too so there is no winning.

I prefer comma separated out of all the options as this makes me 😭 

> -kube-context="ctx-1" -kube-context="ctx-2" -kube-context="ctx-3" -kube-context="ctx-4"-kube-context="ctx-5" -kube-context="ctx-6" -kube-namespace="ns1" -kube-namepsace="ns2" -kube-namepsace="ns3" -kube-namepsace="ns4" -kube-namepsace="ns5"

Reply:
yeah... my example of keeping track with the multiple arguments one maybe isn't the best. The list at least enforces all the "contexts" for instance are right next to each other, you can technically put the flags anywhere. One after the other, between other flags, etc.

Reply:
was talking to Curt, maybe we're at a point where it makes more sense to have a config file for these arguments...

one refactor at a time

Reply:
💯 a config file will make it much better. 

>one refactor at a time, I'll stick with lists for now

So what did you all decide? They are both lists.

Reply:
comma separated lists, sorry for the confusion. I don't think of the multi argument one as a "list input" so I keep referring to the comma separated one as just list 😓 

I realise they both get consumed as lists 😅